### PR TITLE
tests(smoke): update CLS-AF expectation

### PR DESCRIPTION
**Pull request by adamraine**
CLS weighted scores were erroneous and should be fixed in ToT:
https://chromium.googlesource.com/chromium/src/+/042fbfb4cc6a675da0dff4bf3fc08622af42422b

### CODE REVIEW

**Comment by brendankenny** [comment]
Interesting that this affects us. From the change description, at least, I wouldn't expect handling zoom would change things for our smoke test.

**Comment by adamraine** [comment]
This was the full bisect list:
https://chromium.googlesource.com/chromium/src/+log/02c23acd7d6c319ae7976208159e202abd246934..8384f1379a4346fc78da6945a335d947e75632f8

I didn't notice any other suspect changes.

**Comment by brendankenny** [comment]
oh, I think you're definitely right, I'm just trying to understand why our number would change. I have no idea what the larger implications of `PropertyTreeState::Root()` vs `frame.LocalFrameRoot().ContentLayoutObject()->FirstFragment().LocalBorderBoxProperties()` are, though :)

File: lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js [hunk 1]
```diff
@@ -303,13 +303,16 @@ module.exports = [
               {
                 // Weighted CLS score was added to the trace in m90:
                 // https://bugs.chromium.org/p/chromium/issues/detail?id=1173139
-                _minChromiumMilestone: 90,
+                //
+                // Weighted score on emulated mobile bug fixed in m91:
+                // https://chromium.googlesource.com/chromium/src/+/042fbfb4cc6a675da0dff4bf3fc08622af42422b
+                _minChromiumMilestone: 91,
```
Comment (adamraine) [inline lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js:309]:
```suggestion
                //
                // Weighted score on emulated mobile bug fixed in m92:
                // https://chromium.googlesource.com/chromium/src/+/042fbfb4cc6a675da0dff4bf3fc08622af42422b
                _minChromiumMilestone: 92,
```

**Review by brendankenny — APPROVED** [review]
There's some weirdness here that @adamraine, @paulirish, and I looked into a little bit but haven't figured out yet. e.g. when I try out the crbug's demo https://output.jsbin.com/wegixex using m91 and m92 (where the change landed), I'm not getting different CLS `score` or `weightedScoreDelta` in Lighthouse (have to hack around `NO_FCP` errors). If this change implies that our defaults run into a CLS change with m92, I would have expected a change on that page as well, but maybe there's something about the iframe relative sizing that just happens to work out?

However, when we've landed clsAF smoke changes before (#11874 and #12034) I don't think we actually verified the numbers, we just trusted what Chrome gave us. Should we do the same here to unblock CI and file a follow-up issue to make sure we do our due diligence before making all-frame stuff visible in the report?

**Comment by brendankenny** [comment]
> when I try out the crbug's demo https://output.jsbin.com/wegixex using m91 and m92 (where the change landed), I'm not getting different CLS `score` or `weightedScoreDelta` in Lighthouse

@patrickhulce pointed out that the demo page has a `<meta name="viewport" content="width=device-width">` but our smoke test does not, which would affect zooming. Adding a meta viewport to the smoke test makes it so CLS doesn't change from m91 to m92, so that probably solves that mystery.

**Pull request merged**